apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -566,12 +566,20 @@ public void onReset() {
                     Toast.makeText(getActivity(), String.format(getActivity().getString(R.string.target_id_changed_format), defaultTargetId), Toast.LENGTH_SHORT).show();
                 }
             });
-            initEditText(R.string.user_id_key, R.string.user_id_title, USER_ID_KEY, null, new EditTextListener() {
+            initEditText(R.string.user_id_key, R.string.user_id_title, USER_ID_KEY, getString(R.string.user_id_unset), new EditTextListener() {
 
                 @Override
                 public void onSave(String value) {
-                    setKeyValue(USER_ID_KEY, value);
-                    MainActivity.setUserId(value);
+                    if (value != null && value.isEmpty()) {
```

Comment:
Please do not change onSave and what is stored, please rollback this (it was already asked before).
It's only when we display the value that we will use the new texts.

Reply:
Update the fix via add a method to get summary when the userid is null or empty